distribution/lib/Standard/Google_Api/0.0.0-dev/src/Google_Sheets.enso

```diff
@@ -3,15 +3,14 @@ import Standard.Base.Data.Array_Proxy.Array_Proxy
 import Standard.Base.Metadata.Display
 from Standard.Base.Metadata import make_single_choice, Widget
 from Standard.Base.Metadata.Widget import Single_Choice, Vector_Editor
+from Standard.Base.Enso_Cloud.Enso_Secret import as_credential_reference
 
 from Standard.Table import Table
 
-polyglot java import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
-polyglot java import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
-polyglot java import com.google.api.client.json.gson.GsonFactory
-polyglot java import com.google.api.services.sheets.v4.Sheets
-polyglot java import com.google.api.services.sheets.v4.SheetsScopes
-polyglot java import java.util.Collections
+polyglot java import java.io.IOException
+
+polyglot java import org.enso.google.GoogleSheetsHelpers
```

Comment:
Replacing direct `com.google.api.*` imports with fewer, but explicit `org.enso.google` ones is great!